x-pack/test_serverless/shared/config.base.ts

```diff
@@ -119,13 +119,12 @@ export default async () => {
         ])}`,
         // This ensures that we register the Security SAML API endpoints.
         // In the real world the SAML config is injected by control plane.
-        // basic: { 'basic': { order: 0 } },
         `--plugin-path=${samlIdPPlugin}`,
         '--xpack.cloud.id=ftr_fake_cloud_id',
         '--xpack.security.authc.selector.enabled=false',
         `--xpack.security.authc.providers=${JSON.stringify({
-          basic: { basic: { order: 0 } },
-          saml: { 'cloud-saml-kibana': { order: 1, realm: 'cloud-saml-kibana' } },
+          saml: { 'cloud-saml-kibana': { order: 0, realm: 'cloud-saml-kibana' } },
+          basic: { 'cloud-basic': { order: 1 } },
```

Comment:
@azasypkin Oleg, could you share more details around this values and what `order` does here? Just so that we can make changes in the future.

Reply:
Sure, we document `order` here https://www.elastic.co/guide/en/kibana/current/kibana-authentication.html#multiple-authentication-providers. But here, specifically, we want to ensure that SAML is the default authentication method whenever a user navigates to Kibana. In other words, Kibana should attempt to authenticate the user using the provider with the lowest order if the Login Selector is disabled (which is what we'll have in MKI soon). By declaring `cloud-basic` with a higher order, we indicate that basic authentication can still be used in two scenarios:

* When SAML authentication cannot handle the user request.
* When the user enters basic credentials directly from the `/login` page.